src/Rebing/GraphQL/Support/PaginationType.php

```diff
@@ -59,6 +59,18 @@ protected function getPaginationFields($typeName)
                 'resolve'       => function(LengthAwarePaginator $data) { return $data->lastItem(); },
                 'selectable'    => false,
             ],
+            'last_page' => [
+                'type'          => GraphQLType::int(),
+                'description'   => 'The last page (number of pages)',
+                'resolve'       => function(LengthAwarePaginator $data) { return $data->lastPage(); },
+                'selectable'    => false,
+            ],
+            'has_more_pages' => [
+                'type'          => GraphQLType::boolean(),
```

Comment:
This also needs to be wrapped in `GraphQLType::nonNull()`, `hasMorePages` _always_ returns a boolean

Reply:
Ok :)